src/python/ensembl/io/genomio/integrity.py

```diff
@@ -733,10 +732,7 @@ def main() -> None:
     """Main entrypoint."""
     mod = argschema.ArgSchemaParser(schema_type=InputSchema)
 
-    # Start
-    inspector = IntegrityTool(mod.args["manifest_file"])
-    if mod.args.get("brc_mode"):
-        inspector.set_brc_mode(True)
+    inspector = IntegrityTool(mod.args["manifest_file"], mod.args["brc_mode"])
```

Comment:
If br_mode is not provided in the args, this crashes with KeyError. Fix:
```suggestion
    inspector = IntegrityTool(mod.args["manifest_file"], mod.args.get("brc_mode", False))
```

Reply:
This is one of those changes that I missed. The idea is to set the default value of `brc_mode` to false, so it acts as a flag. Will apply it now.